app-office/ledger/ledger-3.2.0.ebuild

```diff
@@ -0,0 +1,115 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit bash-completion-r1 check-reqs cmake python-single-r1
+
+DESCRIPTION="A double-entry accounting system with a command-line reporting interface"
+HOMEPAGE="https://www.ledger-cli.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+IUSE="debug doc python"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+RESTRICT="test"
+
+CHECKREQS_MEMORY=8G
+
+RDEPEND="
+	dev-libs/boost:=[python?]
+	dev-libs/gmp:0=
+	dev-libs/mpfr:0=
+	python? (
+		$(python_gen_cond_dep '
+			dev-libs/boost:=[${PYTHON_USEDEP}]
+			dev-python/cheetah3:=[${PYTHON_USEDEP}]
+		')
+		${PYTHON_DEPS}
+	)
+"
+DEPEND="
+	${RDEPEND}
+	dev-libs/utfcpp
+	doc? (
+		sys-apps/texinfo
+		virtual/texi2dvi
+		dev-texlive/texlive-fontsrecommended
+	)
+"
+
+pkg_pretend() {
+	if use python; then
+		check-reqs_pkg_pretend
+	fi
+}
+
+pkg_setup() {
+	if use python; then
+		check-reqs_pkg_setup
+		python-single-r1_pkg_setup
+	fi
+}
+
+src_prepare() {
+	cmake_src_prepare
+
+	# Want to type "info ledger" not "info ledger3"
+	sed -i -e 's/ledger3/ledger/g' \
+		doc/ledger3.texi \
+		doc/CMakeLists.txt \
+		test/CheckTexinfo.py \
+		tools/cleanup.sh \
+		tools/gendocs.sh \
+		tools/prepare-commit-msg \
+		tools/spellcheck.sh \
+		|| die "Failed to update info file name in file contents"
+
+	mv doc/ledger{3,}.texi || die "Failed to rename info file name"
+
+	rm -r lib/utfcpp || die
+	rm cmake/FindPython.cmake || die
```

Comment:
cmake.eclass provides CMAKE_REMOVE_MODULES_LIST so you don't have to perform that; will fix on merge.

Reply:
Hi! Thanks!

Actually, there's a new version 3.2.1 with some additional fixes to CMake, so my changes to ebuild can be simplified a bit.
- removal of Cmake modules no longer needed (it is checked by upstream CMakeLists.txt)
- DPython_INCLUDE_DIR can be removed from local flags (there was an error in CMakeLists.txt so it was necessary)
- cheetah3 not needed as ledger python server was removed from the tree.